OMEdit/OMEdit/OMEditGUI/Annotations/LineAnnotation.cpp

```diff
@@ -1315,6 +1315,19 @@ void LineAnnotation::duplicate()
   pLineAnnotation->setSelected(true);
 }
 
+void LineAnnotation::updateConnectionOrTransistionOrInitialState(QString const& annotation)
```

Comment:
Maybe we should rename this function to `reDraw` since that is what it is doing.

Reply:
Sounds good. A tad bit shorter 🖌